users/models.py

```diff
@@ -93,3 +93,31 @@ class Apartment(models.Model):
 
     def __str__(self):
         return f"Owner:{self.owner}, Addres:{self.address}, City:{self.city}"
+
+
+class Preferences(models.Model):
+
+    not_smoking = models.BooleanField(blank=True)
+    pets_allowed = models.BooleanField(blank=True)
+    gay_friendly = models.BooleanField(blank=True)
+    air_conditioner = models.BooleanField(blank=True)
+    balcony = models.BooleanField(blank=True)
+    elevator = models.BooleanField(blank=True)
+    long_term = models.BooleanField(blank=True)
+    immediate_entry = models.BooleanField(blank=True)
+
+    class Meta:
+        abstract = True
+
+    def __str__(self):
+        preferences = [field.name for field in self._meta.fields
+                       if field.name != "id" and getattr(self, field.name)]
+        return ", ".join(preferences)
+
+
+class ApartmentProperties(Preferences):
```

Comment:
I'm not sure what this gives you.

Reply:
I Got it, I will change that and push again.